### ISSUE TITLE
Glance and Keyboard Focus

### ISSUE THREAD

**Issue by RemyPorter** [zen-browser/desktop#4687]
### Captchas

- [x] I have read the instructions.
- [x] I have searched existing issues and avoided creating duplicates.
- [x] I am not filing an enhancement request.

### What happened?

When a newly Glanced pane appears, it doesn't have keyboard focus. I'm not entirely certain where keyboard focus is, but it's not on the pane. This means that I can't scroll using the keyboard (arrow keys, pg up/dn), and extensions like Vimium C don't work correctly (since they don't have focus on the pane).

Clicking on the pane resolves this, as it gives the pane focus. This is a minor nuisance, but as a keyboard-first type person, it makes Glance significantly less useful as a feature.

I've confirmed this behavior on both MacOS and Linux.

Additionally, if an extension (like VimC) traps ESC (because I want to dismiss the VimC overlay) the Glance pane also traps it, thus closing the glance. This may be a bug in the extension (because it's not preventing the bubbling of ESC) or it may be with Glance capturing ESC before the page gets it. Ideal behavior would be for the page/extensions to get first grab of ESC.

### Reproducible?

- [x] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.7.2b

### What platform are you seeing the problem on?

Other

### Relevant log output if applicable

```shell

```

**Comment by octaviusz** [comment]
I solved the first problem with loss of focus by adding to Vimium C: `shortcut userCustomized1 command="visitPreviousTab"` strangely enough, this causes the focus to be on the open tab in Glance. And use `Ctrl-[` instead of `ESC`.

**Comment by RemyPorter** [comment]
Hmm, I added that shortcut and it still doesn't work for me.

**Comment by octaviusz** [comment]
Did you add a bind for Custom shortcut 1? (Extensions->Manage Extensions Shortcuts)

![Image](https://github.com/user-attachments/assets/737bde8f-16be-4fc3-88a6-08447e01450c)

**Comment by RemyPorter** [comment]
Ohhh, I see what you meant- I got it in my head that by simply adding a shortcut I was triggering some behavior that would do this automatically. But no, this is just a shortcut that lets me move the focus where I want it- definitely a useful workaround. I just misread and was a dummy.
